google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb

```diff
@@ -1274,6 +1274,9 @@ def external url, format: nil
         #   dashes. International characters are allowed. Label values are
         #   optional. Label keys must start with a letter and each label in the
         #   list must have a different key.
+        # @param [EncryptionConfiguration] destination_encryption_configuration
```

Comment:
I really don't like the idea of adding even more arguments to these methods, but we can't configure this because are yielding the schema and not the job object. An alternative is to create a `LoadJob::Updater` class and yield it instead, similar to what we have done with `Table::Updater`. Would the effort to add an updater object be worth it to not add yet another argument to an already overloaded method?

Reply:
@blowmage Is this something that you could do in a follow-up PR?

Reply:
Yes, but it would have to be made before these changes were released.

Reply:
I started creating a `LoadJob::Updater` class at https://github.com/tswast/google-cloud-ruby/commit/80b23e742e26fdb77cc7a3265435396b392d8823

It has to act a little bit like `Schema` in the same way that `Table::Updater` does since some of the load functions in `Dataset` currently yield a `Schema` but will need to yield a `LoadJob::Updater`.